ibu-imager/postpivot/postpivot.go

```diff
@@ -366,3 +370,20 @@ func (p *PostPivot) setNewClusterID(ctx context.Context, client runtimeclient.Cl
 	}
 	return nil
 }
+
+func (p *PostPivot) applyNMStateFiles(seedReconfiguration *clusterconfig_api.SeedReconfiguration) error {
+	if len(seedReconfiguration.RawNMStateConfigs) == 0 {
+		return nil
+	}
+	nmFile := path.Join(p.workingDir, "nmstate.yaml")
```

Comment:
So single file for all the NMState configs?
So why ask for a list of nmstate configurations in RawNMStateConfigs?

Reply:
If we use apply there is no point to have different files as we just apply it. If we want to drop files in folder (as for me less visible if something is wrong but can be overcome with running validation beforehand) we can have it as dict with name/content .